Makefile

```diff
@@ -95,7 +95,7 @@ edit_etc_hosts:
 create_folders:
 # create some folders to avoid having them owned by root (when created by docker compose)
 	@echo "🥫 Creating folders before docker compose use them."
-	mkdir -p logs/apache2 logs/nginx debug html/data || ( whoami; ls -l . ; false )
+	mkdir -p logs/apache2 debug html/data || ( whoami; ls -l . ; false )
```

Comment:
We must not remove logs/nginx

It might be that we must add create_folders as a dependency to another entry if you have a permission error on that.